repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/serializer/MetricsSerializer4Bears.java

```diff
@@ -0,0 +1,40 @@
+package fr.inria.spirals.repairnator.serializer;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import fr.inria.spirals.repairnator.config.RepairnatorConfig;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.inspectors.metrics4bears.Metrics4Bears;
+import fr.inria.spirals.repairnator.process.inspectors.metrics4bears.MetricsSerializerAdapter4Bears;
+import fr.inria.spirals.repairnator.serializer.engines.SerializedData;
+import fr.inria.spirals.repairnator.serializer.engines.SerializerEngine;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetricsSerializer4Bears extends AbstractDataSerializer {
+
+    public MetricsSerializer4Bears(List<SerializerEngine> engines) {
+        super(engines, SerializerType.METRICS4BEARS);
+    }
+
+    @Override
+    public void serializeData(ProjectInspector inspector) {
+        Gson gson = new GsonBuilder().registerTypeAdapter(Metrics4Bears.class, new MetricsSerializerAdapter4Bears()).create();
+        JsonObject element = (JsonObject)gson.toJsonTree(inspector.getJobStatus().getMetrics4Bears());
+
+        element.addProperty("runId", RepairnatorConfig.getInstance().getRunId());
+        element.addProperty("status",this.getPrettyPrintState(inspector));
+        element.addProperty("stepStatuses", StringUtils.join(inspector.getJobStatus().getStepStatuses(), " -> "));
```

Comment:
Why do you put the status here? It's already serialized with the inspectors.

Reply:
Fixed.